pkg/audit/reporter_test.go

```diff
@@ -53,3 +53,30 @@ func TestReportPendingAudits(t *testing.T) {
 		assert.Equal(t, pending, *pa)
 	})
 }
+
+func TestRecordAuditsAtLeaseOnce(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		err := planet.Satellites[0].Audit.Service.Close()
+		require.NoError(t, err)
+
+		nodeID := planet.StorageNodes[0].ID()
+
+		report := audit.Report{Successes: []storj.NodeID{nodeID}}
+		overlay := planet.Satellites[0].Overlay.Service
+		containment := planet.Satellites[0].DB.Containment()
+
+		// set maxRetries to 0
+		reporter := audit.NewReporter(zap.L(), overlay, containment, 0, 3)
+
+		// expect RecordAudits to try recording at least once
+		failed, err := reporter.RecordAudits(ctx, &report)
+		require.NoError(t, err)
+		assert.Zero(t, failed)
+
+		node, err := overlay.Get(ctx, nodeID)
+		require.NoError(t, err)
+		assert.Equal(t, int64(1), node.Reputation.AuditCount)
```

Comment:
you can use `EqualValues` and you don't have to cast

Reply:
also use require

Reply:
cool, I didn't realize EqualValues does that